src/main/resources/templates/service-status-view.html

```diff
@@ -0,0 +1,77 @@
+{{>layout/header}}
```

Comment:
:wastebasket: This view is not used anymore.

Reply:
removed.